control-plane/grpc/src/operations/watcher/server.rs

```diff
@@ -0,0 +1,73 @@
+use crate::{
+    misc::traits::ValidateRequestTypes,
+    operations::watcher::traits::WatcherOperations,
+    watcher,
+    watcher::{
+        get_watches_reply,
+        watcher_grpc_server::{WatcherGrpc, WatcherGrpcServer},
+        GetWatchesReply, GetWatchesRequest, WatchReply,
+    },
+};
+use std::sync::Arc;
+use tonic::Response;
+
+/// RPC Watcher Server
+#[derive(Clone)]
+pub struct WatcherServer {
+    /// Service which executes the operations.
+    service: Arc<dyn WatcherOperations>,
+}
+
+impl WatcherServer {
+    /// returns a new WatcherServer with the service implementing Watcher operations
+    pub fn new(service: Arc<dyn WatcherOperations>) -> Self {
+        Self { service }
+    }
+    /// coverts the replicaserver to its corresponding grpc server type
```

Comment:
Nit: Typo "converts". Also `replicaserver` should be `WatcherServer`.

Reply:
Done